lxdock/conf/config.py

```diff
@@ -115,6 +153,27 @@ def _get_container_config_dict(self, container_dict):
         del container_config['containers']
         return container_config
 
+    def _get_dotenv_items(self, env_filepath):
```

Comment:
Or we could use something like https://github.com/theskumar/python-dotenv and add it to the project's dependencies.